src/views/Feed/Feed.js

```diff
@@ -16,18 +16,18 @@ import FeedSummary from './FeedSummary'
 import FeedUser from './FeedUser'
 
 function Feed(props) {
-  const { children, className, items, size } = props
+  const { children, className, events, size } = props
   const classes = cx('ui', className, size, 'feed')
   const rest = getUnhandledProps(Feed, props)
 
-  if (!items) {
+  if (!events) {
     return <div {...rest} className={classes}>{children}</div>
   }
 
-  const eventsJSX = items.map(({ key, event }, index) => {
+  const eventsJSX = events.map(({ key, ...eventData }, index) => {
     const finalKey = key || index
```

Comment:
Ah, I see the example now.  I'm working on the Modal updates right now, almost done. Then, I'll pull the Feed down and come up with some suggestions.